### PR TITLE
Fix gd CallMap for PHP 8

### PR DESCRIPTION
**Pull request by iluuu1994**

### CODE REVIEW

File: dictionaries/CallMap.php [hunk 1]
```diff
@@ -5335,117 +5335,116 @@
 'iis_start_service' => ['int', 'service_id'=>'string'],
 'iis_stop_server' => ['int', 'server_instance'=>'int'],
 'iis_stop_service' => ['int', 'service_id'=>'string'],
-'image2wbmp' => ['bool', 'im'=>'resource', 'filename='=>'?string', 'threshold='=>'int'],
```
Comment (iluuu1994) [inline dictionaries/CallMap.php:5338]: Was removed in PHP 8. https://wiki.php.net/rfc/image2wbmp

**Review by weirdan — COMMENTED** [review]
This needs corresponding changes to callmap delta.

**Comment by iluuu1994** [comment]
The parameter names for PHP 7.4 might not match. Not sure if that matters.

**Comment by weirdan** [comment]
> The parameter names for PHP 7.4 might not match. Not sure if that matters.

I believe it shouldn't.

**Comment by iluuu1994** [comment]
Anything holding this up?

**Comment by muglug** [comment]
Nope, thanks!

**Pull request merged**